tsconfig.json

```diff
@@ -4,6 +4,8 @@
     "forceConsistentCasingInFileNames": true,
     "moduleResolution": "node",
     "strict": true,
-    "skipLibCheck": true
-  }
+    "skipLibCheck": true,
+    "types": ["node"]
```

Comment:
This makes it explicit about which types are implicitly included (to stop Jest types being loaded).